container.go

```diff
@@ -108,15 +112,55 @@ type State struct {
 	FinishedAt time.Time `json:"FinishedAt,omitempty" yaml:"FinishedAt,omitempty"`
 }
 
-// String returns the string representation of a state.
+// String returns a human-readable description of the state
 func (s *State) String() string {
 	if s.Running {
 		if s.Paused {
```

Comment:
Is there never a case where `s.Paused && !s.Running`?, Similar question for `Restarting`, and the assumption that the fallthrough is `exited.`